test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -22,14 +22,16 @@
  */
 
 /* @test
- * @bug 8011536 8151430
+ * @bug 8011536 8151430 8316304
  * @summary Basic test for creationTime attribute on platforms/file systems
  *     that support it.
  * @library  ../.. /test/lib
+ * @modules java.base/sun.nio.fs:open
  * @build jdk.test.lib.Platform
  * @run main CreationTime
  */
 
+import java.lang.reflect.Method;
```

Comment:
Instead of using reflective access, would it be possible to use the "os.version" property and a minimum kernel version (4.11?)?

```
jshell> System.getProperty("os.version")
$1 ==> "5.19.0-50-generic"

jshell> 
bpb@bpb-vb-u2204:~$ uname -a
Linux bpb-vb-u2204 5.19.0-50-generic #50-Ubuntu SMP PREEMPT_DYNAMIC Mon Jul 10 18:24:29 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux```

Reply:
It's not the full story. On an alpine build, on `5.19.0-50` it would report the wrong value. It's not supported, since the libc impl doesn't support it. The kernel itself would as a regular, non-alpine build, attests. I could use `!Platform.isMusl()` in addition if that sounds OK?

Reply:
Hmm, @bplb `Platform.isMusl()` doesn't seem to work reliably. Running an alpine binary on Linux (Fedora) in my case, breaks this test as it gets `Platform.isMusl() == false`. Also, there is a chance where the JDK runs on an old glibc (<2.28) and new enough kernel, where this wouldn't work either. Using the reflective approach seems most consistent with what the code is actually doing. More thoughts?

Reply:
For now I'd like to keep the reflective approach until we know a better way to determine it.